src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,376 @@
+
+-   The target frame of reference is the implicit coordinate system of the transform.
+
+-   The target frame of reference has the same units as the `PixelSizeUnits` metadata.
+
+-   The chunk transform is described by 2 metadata fields: a transformation matrix and a
+    description of the axis of the matrix.
+
+-   Other transformations should be described in derivatives.
+
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Key name**                  | **Requirement level**                                            | **Data type**                          | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ----------------------------- | ---------------------------------------------------------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ChunkTransformationMatrix     | OPTIONAL                                                         | [array][] of [arrays][] of [numbers][] | 3x3 or 4x4 matrix describing spatial chunk transformation, for 2D and 3D respectively (for examples: `[[2, 0, 0], [0, 3, 0], [0, 0, 1]]` in 2D for 2x and 3x scaling along the first and second axis respectively or `[[1, 0, 0, 0], [0, 2, 0, 0], [0, 0, 3, 0], [0, 0, 0, 1]]` in 3D for 2x and 3x scaling along the second and third axis respectively). Note that non-spatial dimensions like time and channel are not included in the transformation matrix. |
+| ChunkTransformationMatrixAxis | OPTIONAL, but REQUIRED if `ChunkTransformationMatrix` is present | [arrays][] of [strings][]              | Describe the axis of the ChunkTransformationMatrix (for examples: `['X', 'Y']` or `['Z', 'Y', 'X']`)                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
In that case the XYZ is meant to refer to the data array indices.

Our issue with applying the matrix “to the pixel indices in the original order” is that in the case of OME-TIFF, The DimensionOrder includes dimention T (time) and C (channel) as well, in any order, so T and C could be mixed with X, Y and Z.

In the case of PNG, I’m not sure what the “original” order would refer to. Unless I’m mistaken, even knowing that X is the width and Y is the height, when loading a 2D image as a matrix, the first indices are the rows (Y) and the second one the columns (X), which can be confusing as well.

So we chose to add the `ChunkTransformationMatrixAxis` to reduce the complexity of having to keep track of the original dimension order, as well as implementing non-spatial dimension into the matrix.

I understand your point that this might be too much flexibility and I am not opposed to making changes here but could use some help figuring this out. Tagging @satra  and @LeeKamentsky here if they want to chip in.

Would it be possible to impose an order (similar to [AnatomicalLandmarkCoordinates](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#anatomical-landmarks) in MRI) which is always XYZ?

Reply:
I was actually implementing this today and it's easy to pick up the indices from ChunkTransformationMatrixAxis and then apply them to ChunkTransformationMatrix to reorder the matrix the way you like it. So I am fine with the added flexibility and I agree with @mariehbourget that it future-proofs us for datasets with dimensions such as channel and time.